routemaster/cli.py

```diff
@@ -19,17 +20,24 @@
 @click.option(
     '-c',
     '--config-file',
+    'config_files',
     help="Path to the service config file.",
     type=click.File(encoding='utf-8'),
     required=True,
+    multiple=True,
 )
 @click.pass_context
-def main(ctx, config_file):
+def main(ctx, config_files):
     """Shared entrypoint configuration."""
     logging.getLogger('schedule').setLevel(logging.CRITICAL)
 
+    config_data = layer_loader.load_files(
+        config_files,
+        loader=yaml.load,
+    )
```

Comment:
Do we think it's worth breaking out a util which wraps this?

Unfortunately it's not trivial to do this as while many usages operate on paths, this usage (the main CLI entry point) operates on file objects (currently). We could modify this usage to expected `click.Path` rather than `click.File`, but that's a larger change than I have in mind for this PR.

Reply:
It would be nice to have a util for it, but I don't think it should block this PR.